df/plugin.go

```diff
@@ -412,14 +422,18 @@ func (dfs *dfStats) collect(procPath string, excluded_fs_names []string, exclude
 		dfm.Filesystem = rightFields[1]
 		dfm.FsType = rightFields[0]
 		dfm.UnchangedMountPoint = leftFields[4]
-		if leftFields[4] == "/" {
-			dfm.MountPoint = "rootfs"
+		if keep_original_mountpoint {
```

Comment:
I thought the default behavior would have kept the original mount point now that the framework supports namespace elements containing `/` characters.  

Was this an effort to be conservative?  

I'd be inclined to change the behavior to keep the original mount point (and not provide an option preserving the old behavior) or at least make the default behavior preserve the  mount point.